src/app/components/Table/TableHeader/TableHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import TableCell from '../TableCell/TableCell';
+import styles from './TableHeader.module.scss';
+
+const TableHeader = ({ cellData }: { cellData: ReadonlyArray<string> }) => {
+  return (
+    <div className={styles.header}>
+      {cellData.map((cellValue) => (
+        <TableCell cellValue={cellValue} key={cellValue} />
```

Comment:
It will be unique?

Reply:
It should be uniq as it is name of propertyes in data. Also I use mock array for names (in current version). Late it will be refactored